source/html/renderer/src/card-elements.ts

```diff
@@ -839,6 +839,7 @@ export class ToggleInput extends Input {
 export class Choice {
     title: string;
     value: string;
+    isSelected: boolean;
```

Comment:
This is not the design. To pre-select a choice, set the Choice input's value property to one of it's choices value. This is the value pre-selection/default value is done for all input types.